src/Microsoft.Health.Fhir.Api/Features/Routing/RouteNames.cs

```diff
@@ -65,6 +65,8 @@ internal static class RouteNames
 
         internal const string PurgeHistoryDefinition = "PurgeHistoryDefinition";
 
+        internal const string SearchParameterStatusOperationDefinition = "SearchParameterStatusOperationDefinition";
```

Comment:
Low priority: You can replace the static string with nameof(SearchParameterStatusOperationDefinition).

Reply:
I can add that as a fix in another PR.